packages/desktop-client/src/components/accounts/TransactionsTable.test.js

```diff
@@ -599,7 +577,6 @@ describe('Transactions', () => {
   test('dropdown selects an item when clicking', async () => {
     const { container, getTransactions } = renderTransactions();
 
-    let input = editField(container, 'category', 2);
```

Comment:
It looks like this has side effects, maybe keep it but just remove the `let input =` part?

Reply:
Good catch https://github.com/actualbudget/actual/pull/421/commits/b5d34b9d11082d4b1d1afb63623966935ef77fbc